person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1000..9999)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @parent_permission || of_age?
```

Comment:
- [ ] of_age? is a method you are passing and I think it should have parethensis: `of_age?( )`

Reply:
I think `of_age?` method doesn't need parentheses since it doesn't take any arguments. Also, even if I add the parentheses, Rubocop linter removes them.